util/Migrator/DbScripts/2023-12-04_00_GrantIndexes.sql

```diff
@@ -0,0 +1,42 @@
+IF COL_LENGTH('[dbo].[Grant]', 'Id') IS NULL
+    BEGIN
+        ALTER TABLE [dbo].[Grant]
+            ADD [Id] INT NOT NULL IDENTITY
+
+        ALTER TABLE [dbo].[Grant]
+            DROP CONSTRAINT [PK_Grant];
+
+        ALTER TABLE [dbo].[Grant]
+            ADD CONSTRAINT [PK_Grant] PRIMARY KEY CLUSTERED ([Id] ASC);
+
+        CREATE UNIQUE INDEX [IX_Grant_Key]
+            ON [dbo].[Grant]([Key])
+            WHERE [Key] IS NOT NULL;
+    END
```

Comment:
These need to be done online, right?

Reply:
After my experience last night, online may not be enough, can rewrite this to create a grant_new table, copy data over, create indexes and then rename grant to grant_old and grant_new to grant. It is an extremely busy table and even the slightest interruption can cause a large backlog of queries

Reply:
I think this PR should wait until this is done for cloud: https://github.com/bitwarden/server/pull/3443/files

Then I dont think we have the table business problems.